src/test/java/org/jenkins/plugins/lockableresources/BasicIntegrationTest.java

```diff
@@ -0,0 +1,54 @@
+package org.jenkins.plugins.lockableresources;
+
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Semaphore;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.Issue;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.MockBuilder;
+import org.jvnet.hudson.test.TestExtension;
+
+import hudson.Launcher;
+import hudson.model.AbstractBuild;
+import hudson.model.BuildListener;
+import hudson.model.FreeStyleBuild;
+import hudson.model.FreeStyleProject;
+import hudson.model.Result;
+
+public class BasicIntegrationTest {
+
+	@Rule
+	public JenkinsRule j = new JenkinsRule();
+
+	@Test
+	@Issue("JENKINS-34853")
+	public void security170fix() throws Exception {
```

Comment:
I guess this fails when the `src/main/` part is reverted and the test is run with a sufficiently new `jenkins.version`?